File: demoMavenJava/src/main/java/com/example/demo/DemoApplication.java
package com.example.demo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
@RestController
public class DemoApplication {

        public static void main(String[] args) {
                SpringApplication.run(DemoApplication.class, args);
        }

        @GetMapping("/devs13")
        public String devs13(@RequestParam(value = "name", defaultValue = "Equipe em Desenvolvimento com Spring Boot") String name) {
                return String.format("End point ativo %s", name);
        }

}

File: maven/kotlin/demo/src/main/kotlin/com/example/demo/DemoApplication.kt
package com.example.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController


@SpringBootApplication
@RestController
class DemoApplication

fun main(args: Array<String>) {
	runApplication<DemoApplication>(*args)
}

@GetMapping("/devs13")
  fun demo(model: Model): String {
    model["title"] = "Blog"
    return "Usando Kotlin com Spring Boot"
  }